app/src/main/java/com/xinto/opencord/ui/component/OCAsyncImage.kt

```diff
@@ -25,6 +25,7 @@ fun OCAsyncImage(
     url: String?,
     modifier: Modifier = Modifier,
     size: Size = Size.ORIGINAL,
+    loadingEnabled: Boolean = true,
```

Comment:
I don't see a reason for loadingEnabled, since everything that's loading should display a shimmer, doesn't matter if it's a placeholder or an actual image loading

Reply:
Considering it fills max size, its essentially a huge flashbang. Not very pretty

Reply:
Also stock doesn't have this + its a lot more neat when it just pops up when loaded.
Especially on devices with slower internet

Reply:
> Considering it fills max size, its essentially a huge flashbang. Not very pretty

Then maybe it could just be a onSurface box without the shimmering for images. 

> Also stock doesn't have this + its a lot more neat when it just pops up when loaded. Especially on devices with slower internet

I don't see how it's related to bandwidth at all. Making it just pop up randomly can be annoying as content shifts when it appears out of nowhere.